docs/development/development.md

```diff
@@ -3,6 +3,36 @@
 
 This document helps you get started developing Dapr CLI. If you find any problem while following this guide, please create a Pull Request to update this document.
 
+# Developing in a VS Code Dev Container
+
+If using [VS Code](https://code.visualstudio.com/), you can build and debug the Dapr CLI from a [Dev Container](https://code.visualstudio.com/docs/remote/containers) with very little initial setup on your machine.
+
+## Prerequisites
+
+ - [Docker](https://www.docker.com/products/developer-tools)
+ - [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) VS Code extension
+
+## Setup
+
+1. Clone the repository:
+
+    ```bash
+    $ git clone https://github.com/dapr/cli.git
+    ```
+
+1. Open the repository in VS Code

```

Comment:
Enumeration is always "1". Is it correct?

More importantly, can you be more specific in the instructions for the las 2 steps here. If not with commands, then with instruction on UI buttons/menus or screenshots.

Reply:
markdown automatically increases the number :)